ansible-collection/examples/cloud_credential/create.yaml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Can you please include the playbook for all the other CRUD operations as well?

Reply:
Added all playbooks